obspy/io/sac/core.py

```diff
@@ -470,5 +474,7 @@ def _internal_write_sac(trace, buf, byteorder="<", **kwargs):
     else:
         msg = "Invalid byte order. It must be either '<', '>', 0 or 1"
         raise ValueError(msg)
-    sac = SACTrace.from_obspy_trace(trace)
+
+    keep_sac_header = kwargs.get('keep_sac_header', True)
```

Comment:
Any reason why not to properly expose this `kwarg` in the function definition, instead of keeping it in partial obscurity in `kwargs`. That also makes the default value clear right away

```
def _internal_write_sac(trace, buf, byteorder="<", keep_sac_header=True, **kwargs):
```

Reply:
I made this change and rebased on `master` again.   I hope that was the right thing to do.